src/helpers/date-functions.ts

```diff
@@ -41,3 +41,32 @@ export const dateToCurrentTimeDate = (endDateString: string): string => {
   now.setFullYear(endDate.getFullYear());
   return now.toISOString();
 };
+
```

Comment:
Should this not be in some other place? We put date related functions here.

Reply:
I misread date with data:d 
Sure, I will move it to another place